<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>X2Dome</class>
 <widget class="QDialog" name="X2Dome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>760</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>386</width>
    <height>760</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>386</width>
    <height>760</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NexDome controller settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="X2Form" name="X2Form">
     <property name="minimumSize">
      <size>
       <width>362</width>
       <height>524</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1024</width>
       <height>900</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="SerialPortName" stdset="0">
      <string/>
     </property>
     <widget class="QGroupBox" name="MaxDomeIIParams">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>104</y>
        <width>336</width>
        <height>225</height>
       </rect>
      </property>
      <property name="title">
       <string>NexDome Params</string>
      </property>
      <widget class="QCheckBox" name="homeOnUnpark">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>193</y>
         <width>140</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Home after unpark</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>29</y>
         <width>281</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Calibrate</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="hasShutterCtrl">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>151</y>
         <width>183</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Shutter controller present</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="needReverse">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>130</y>
         <width>132</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Reverse direction</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="homeOnPark">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>172</y>
         <width>136</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Home before park</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>64</y>
         <width>160</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Home Position (Deg.) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="homePosition">
       <property name="geometry">
        <rect>
         <x>184</x>
         <y>64</y>
         <width>88</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <double>359.990000000000009</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>32</x>
         <y>88</y>
         <width>144</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Park Postition (Deg.) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="parkPosition">
       <property name="geometry">
        <rect>
         <x>184</x>
         <y>88</y>
         <width>88</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <double>359.990000000000009</double>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>336</y>
        <width>336</width>
        <height>352</height>
       </rect>
      </property>
      <property name="title">
       <string>Controller Informations and settings</string>
      </property>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>64</x>
         <y>200</y>
         <width>152</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Dome battery cutoff :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="lowRotBatCutOff">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>200</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="value">
        <double>11.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>64</x>
         <y>224</y>
         <width>152</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Shutter battery cutoff :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="lowShutBatCutOff">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>224</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="value">
        <double>11.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="shutterBatteryLevel">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>272</y>
         <width>56</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>12.25 V</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>72</x>
         <y>272</y>
         <width>144</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Shutter battery level :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>56</x>
         <y>67</y>
         <width>160</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Rotation speed (step/s) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="rotationSpeed">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>67</y>
         <width>104</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="rotationAcceletation">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>100</y>
         <width>104</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>100</y>
         <width>200</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Rotation acceleration (step/s) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>34</y>
         <width>208</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Number of steps per revolution :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="ticksPerRev">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>34</y>
         <width>104</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>128</x>
         <y>320</y>
         <width>88</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Rain sensor :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="rainStatus">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>320</y>
         <width>96</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>64</x>
         <y>296</y>
         <width>152</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Pointing Error (Deg.) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="domePointingError">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>296</y>
         <width>48</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="shutterAcceleration">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>166</y>
         <width>104</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>166</y>
         <width>200</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Shutter acceleration (step/s) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>248</y>
         <width>136</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Dome battery level :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="domeBatteryLevel">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>248</y>
         <width>56</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>12.25 V</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>56</x>
         <y>133</y>
         <width>160</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Shutter speed (step/s) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="shutterSpeed">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>133</y>
         <width>104</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_logo">
      <property name="geometry">
       <rect>
        <x>96</x>
        <y>8</y>
        <width>168</width>
        <height>64</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>NexDome.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="X2_PhotoFileName" stdset="0">
       <string>NexDome.png</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>96</x>
        <y>64</y>
        <width>168</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>© RTI-Zone.org, 2018</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>128</x>
        <y>696</y>
        <width>216</width>
        <height>40</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonOK">
         <property name="text">
          <string>OK</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>X2Form</class>
   <extends>QFrame</extends>
   <header>x2form.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>8</number>
  </property>
  <property name="gridDeltaY">
   <number>8</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
